<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_IPaddress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>IP Address：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="UserList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="IPaddressChoosen">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getIPAddressButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}</string>
   </property>
   <property name="text">
    <string>Get IPAddress</string>
   </property>
  </widget>
  <widget class="QLabel" name="OnlineUser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Online User：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>560</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>560</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Pic/BKSM.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="openServerButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeServerButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="Designedby">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>465</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Designed by Young Inc © 2021 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Port：</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_IPaddress</zorder>
  <zorder>UserList</zorder>
  <zorder>IPaddressChoosen</zorder>
  <zorder>getIPAddressButton</zorder>
  <zorder>OnlineUser</zorder>
  <zorder>openServerButton</zorder>
  <zorder>closeServerButton</zorder>
  <zorder>Designedby</zorder>
  <zorder>portEdit</zorder>
  <zorder>label_port</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
